implementation/src/main/java/io/smallrye/mutiny/helpers/spies/MultiOnCancellationSpy.java

```diff
@@ -0,0 +1,22 @@
+package io.smallrye.mutiny.helpers.spies;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.subscription.MultiSubscriber;
+
+public class MultiOnCancellationSpy<T> extends MultiSpyBase<T> {
+
+    MultiOnCancellationSpy(Multi<? extends T> upstream) {
+        super(upstream);
+    }
+
+    @Override
+    public void subscribe(MultiSubscriber<? super T> downstream) {
+        upstream.onCancellation().invoke(this::incrementInvocationCount)
+                .subscribe().withSubscriber(downstream);
+    }
+
+    @Override
+    public String toString() {
+        return "MultiOnCancellationSpy{} " + super.toString();
```

Comment:
Are you sure of the format? {} followed by super?

Reply:
IntelliJ-generated, provides a good insight.